Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -137,6 +152,15 @@ public static bool ReportFileExist(string reportFolderPath)
 						return true;
 					}
 				}
+				else
+				{
+					var fileName = Path.GetFileName(Path.GetDirectoryName(reportFolderPath));
+					if (!string.IsNullOrEmpty(fileName) && fileName.Contains("ProjectFiles"))
+					{
+						fileName = fileName.Substring(0, fileName.LastIndexOf(".ProjectFiles", StringComparison.Ordinal));
```

Comment:
For getting the filename without the extension: [Path.GetFileNameWithoutExtension(filename)](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getfilenamewithoutextension?view=netcore-3.1)

Reply:
I've updated the code to use  Path.GetFileNameWithoutExtension(fileName)